src/core/api/link_raw_api.cpp

```diff
@@ -67,6 +84,9 @@ otError otLinkRawSetPanId(otInstance *aInstance, uint16_t aPanId)
     VerifyOrExit(aInstance->mLinkRaw.IsEnabled(), error = OT_ERROR_INVALID_STATE);
 
     otPlatRadioSetPanId(aInstance, aPanId);
```

Comment:
I didn't put them within `LinkRaw` because those functions are not always necessary under the current design. `ncp_base` would sometimes call versions in `link_api.cpp`.